pep-9999.rst

```diff
@@ -205,7 +205,8 @@ Software distributors who have a non-Python-specific package manager
 that manages libraries in the ``sys.path`` of their Python package
 should, in general, ship a ``EXTERNALLY-MANAGED`` file in their standard
 library directory. For instance, Debian may ship a file in
-``/usr/lib/python3.*/EXTERNALLY-MANAGED`` consisting of something like
+``/usr/lib/python3/dist-packages/EXTERNALLY-MANAGED`` consisting of
```

Comment:
I don't think this is right - we wanted the file to be in the `stdlib` directory, not in any of the target packages directory. On buster:
```
$ python3
>>> import sysconfig
>>> sysconfig.get_path("stdlib")
'/usr/lib/python3.7'
```

Reply:
Aha.

So, it's flagging the interpreter, not the `sys.path` setup.